src/main/java/thePackmaster/patches/clawpack/ClawUsePatch.java

```diff
@@ -0,0 +1,25 @@
+package thePackmaster.patches.clawpack;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpireReturn;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.defect.GashAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.blue.Claw;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.clawpack.ClawStrengthPower;
+import thePackmaster.util.Wiz;
+
+@SpirePatch(
+        clz = GashAction.class,
+        method = "update"
+)
+public class ClawUsePatch {
+        public static SpireReturn Postfix(GashAction __instance) {
+            Wiz.applyToSelf(new ClawStrengthPower(Wiz.p(), __instance.amount));
+            return SpireReturn.Return();
```

Comment:
Okay, I ended up going with something very similar to this.  GashAction is prefix'd to go also run an identical process on any CLAW-tagged cards, excepting actual Claw instances since they are covered by the Gash Action.  Other cards that do Sharpen just run the custom version and does not except Claws.  This is necessary because the GashAction receives a card input and modifies it directly, and not all the sources of Sharpen are specifically on a card source.

Additionally the Ghost Claw now inherits previous Sharpen effects, which is explained on the keyword for Sharpen.

ClawStrengthPower is dead.